client/analytics/ad-tracking.js

```diff
@@ -99,25 +114,40 @@ function loadTrackingScripts( callback ) {
 }
 
 function retarget() {
-	if ( ! hasLoadedScripts ) {
+	if ( ! hasFetchedScripts ) {
 		return loadTrackingScripts( retarget );
 	}
 
 	if ( ! retargetingInitialized ) {
 		debug( 'Retargeting initialized' );
 
-		window._fbq.push( [ 'track', 'PixelInitialized', {} ] );
+		window.fbq( 'track', 'PageView' );
 		retargetingInitialized = true;
 	}
 }
 
+function recordAddToCart( cartItem ) {
+	if ( ! hasFetchedScripts ) {
+		return loadTrackingScripts( recordAddToCart.bind( null, cartItem ) );
+	}
+
+	debug( 'Recorded that this item was added to the cart', cartItem );
+
+	window.fbq(
+		'track',
+		'AddToCart',
+		{
+			product_slug: cartItem.product_slug,
+			free_trial: Boolean( cartItem.free_trial )
+		}
+	)
+}
+
 function recordPurchase( product ) {
 	let type;
 
-	if ( ! hasLoadedScripts ) {
-		return loadTrackingScripts( function() {
-			recordPurchase( type );
```

Comment:
This should have been `recordPurchase( product )`, and is fixed in the diff. This almost certainly didn't cause any bugs, because it shouldn't be possible for the user to get in a state where `recordPurchase` is called but the scripts haven't been fetched, because of prior calls to `retarget`.